quint/src/util.ts

```diff
@@ -29,3 +30,17 @@ import JSONbig from 'json-bigint'
 export function unreachable(object: never): never {
   throw new Error(`impossible: non-exhuastive check should fail during type checking ${JSONbig.stringify(object)}`)
 }
+
+/**  A wrapper around lodash zip that ensures all zipped elements are defined
```

Comment:
I will use this! Thanks

Reply:
Ah, I noticed this was moved, but regardless, I didn't know/remember we have this :smile:

Reply:
It was added in 99910e3f3d5523410b92d56ee6106546435c9793 :)